src/debugAdapter/goDebug.ts

```diff
@@ -995,49 +995,67 @@ class GoDebugSession extends DebugSession {
 	protected variablesRequest(response: DebugProtocol.VariablesResponse, args: DebugProtocol.VariablesArguments): void {
 		verbose('VariablesRequest');
 		let vari = this._variableHandles.get(args.variablesReference);
-		let variables;
+		let variablesPromise: Promise<DebugProtocol.Variable[]>;
+		let loadChildren = async (exp: string, v: DebugVariable) => {
```

Comment:
nit: this can be a const.

Reply:
actually weird that tslint doesnt warn on this

Reply:
oh ok, I looked at the tslint.config, it seems some very old config, I guess we can make another PR to update that

Reply:
Will mark as const.